README.md

```diff
@@ -39,6 +39,11 @@ If you have a question about something you can not find in our documentation, th
 
 If you would like to contribute to our documentation see the [website](/website) directory for more information.
 
+## 🗞 Blog Posts
+
+* [Introducing GraphQL Kotlin Client](https://medium.com/expedia-group-tech/introducing-graphql-kotlin-client-b32dc3061a6f)
+* [🎉 Announcing graphql-kotlin 2.0!](https://medium.com/expedia-group-tech/graphql-kotlin-2-0-4006ea41f774)
```

Comment:
```suggestion
## 🗞  Blog Posts and Videos

You can view a collection of our public blog posts and videos on our [page in our docs site](https://expediagroup.github.io/graphql-kotlin/docs/blogs-and-videos). These include release announcements, conference talks about the library, and general talks about GraphQL at Expedia Group. 
```

Reply:
My suggested text:

The [Blogs and videos page](https://expediagroup.github.io/graphql-kotlin/docs/blogs-and-videos) in the GraphQL Kotlin documentation links to blog posts, release announcements, conference talks about the library, and general talks about GraphQL at Expedia Group.

Reply:
...and then of course we need to add these two blog post links to that page...

Reply:
Those posts are already linked, I updated my suggestion, thanks @EXPErdelfavero !

Reply:
I've reversed the order, so that newest posts are on top, and updated the readme as per suggestions.